### ISSUE TITLE
Support Pixel Fold and Pixel Tablet

### ISSUE THREAD

**Issue by connyduck** [tuskyapp/Tusky#3715]
I received the following email from Google Play:

<details>
  <summary>Mail</summary>
  
Hi Tusky for Mastodon,

**TL;DR:** In recent testing of Tusky for Mastodon on the Pixel Tablet and Pixel Fold, we discovered that your app experience is not optimized for large screen devices. We are reaching out to inform you that on the Pixel Fold and Pixel Tablet your app will receive overrides to fix the broken behaviors (more details below). Please reach out to us at android-devrel-ops@google.com if you’ve already fixed your app, would like to opt out of overrides, or would like to learn more about resolving these issues natively.

Recently we tested your Tusky for Mastodon app on the Pixel Tablet and Pixel Fold and found that the app experience is not optimized for large screens. Ideally your app will be Tier 2 large screen optimized, which ensures it will offer a strong user experience by taking advantage of the larger screen real estate with no continuity or layout issues. You can achieve Tier 2 optimization by following our [developer guidance](https://developer.android.com/docs/quality-guidelines/large-screen-app-quality#large_screen_optimized).

We understand reaching Tier 2 may not be feasible in a short timeframe for each app. To give you more time, we will temporarily place your app in compat mode by applying overrides to certain broken behaviors. Your app will have these overrides on the Pixel Tablet and/or Pixel Fold only and will be available to users when devices are released publicly.

More details on the specific overrides applied to your app are provided below, along with guidance on how to disable overrides for debugging purposes, how to resolve issues natively, and different ways you can test your app. **If you have already fixed your app, would like to opt out of overrides, or learn more about our overrides, reach out to us at android-devrel-ops@google.com.**

**Why the app needed overrides:** Content cropped at both sides in Folded portrait(folded)..

**Overrides applied to: com.keylesspalace.tusky**

- **Override Id: 180326845** 
  - Override name: OVERRIDE_MIN_ASPECT_RATIO_MEDIUM
  - What the override does if it's enabled: Changes the min aspect ratio to 3:2
  - Technical details: This change id sets the activity's min aspect ratio to a medium value as defined by OVERRIDE_MIN_ASPECT_RATIO_MEDIUM_VALUE in ActivityInfo which is currently set to 3:2
  - How apps can achieve the same behavior as the override: Set android:minAspectRatio at the activity or app level
  - How to resolve app issue natively (DevRel guidance): You should not set aspect ratio restrictions in your app and make sure you can support different screen sizes. One tool that will help you is leveraging WindowSizeClass: with this, you can support different layouts based on the amount of space your app has on the screen. See more at [goo.gle/window-size-class-compose](https://goo.gle/window-size-class-compose) and [goo.gle/window-size-class-views](https://goo.gle/window-size-class-views)
  - How apps can disable or opt-out of the override: Specify your own aspect ratio restriction or contact us to disable
  - Property flags to adjust override (if applicable): Developer can specify a min aspect ratio. Contact us if you'd like the override removed.
  - adb commands:      
    - To apply the override, run: adb shell am compat enable OVERRIDE_MIN_ASPECT_RATIO_MEDIUM
    - To remove the override, run: adb shell am compat disable OVERRIDE_MIN_ASPECT_RATIO_MEDIUM
- **PDF with before/after screenshots:** Please review the attached PDF to see before and after changes

**Ways to test your app**
To ensure the best user experience for your app, you should test on large screens without overrides and resolve issues natively. If you do not currently have access to a Pixel Tablet or Pixel Fold device, there are a few ways you can still test your app:  • Firebase Test Lab: Email us if interested in testing your app on production devices hosted in our Google data centers.

 • Android Studio emulator: create a Pixel Tablet or Pixel fold emulator using [Android Emulator](https://developer.android.com/studio/run/managing-avds)

If you have any questions around our overrides process, have fixed your app, or would like to opt out of overrides, please reach out to us at android-devrel-ops@google.com.

Thanks,

Android Developer Relations
</details>

with this attached:
[Tusky.pdf](https://github.com/tuskyapp/Tusky/files/11656165/Tusky.pdf)

Looks like Google is trying really hard to make the launch of their new devices a success and even applies overrides for apps that don't work well. Anyway, I think we should fix this on our side as well.

**Comment by connyduck** [comment]
fixed by https://github.com/tuskyapp/Tusky/pull/4689
